### DISCUSSION TITLE
There are still many untranslated parts when setting it to Simplified Chinese language

### DISCUSSION THREAD

**Discussion by xnddxndd** [dbeaver/dbeaver#21049] [Q&A] [answered]
We noticed that when setting it to Simplified Chinese language, there are still many untranslated parts. How can we translate them?

Can we set up the language pack ourselves? Or we can submit the translated text.

**Comment by E1izabeth** [comment, marked as answer]
You can contribute translation. See this guide https://github.com/dbeaver/dbeaver/wiki/Localization

**Comment by xnddxndd** [comment]
Thinkyou，I find it, and understand。tks。

> You can contribute translation. See this guide https://github.com/dbeaver/dbeaver/wiki/Localization

Thinkyou，I find it, and understand。tks。
